src/prefect/concurrency/asyncio.py

```diff
@@ -37,6 +37,7 @@ async def concurrency(
     occupy: int = 1,
     timeout_seconds: Optional[float] = None,
     create_if_missing: bool = True,
+    active: bool = False,
```

Comment:
we only need this if we continue to lean into the implicit creation of limits IMO - if we do decide to keep this I'd advocate for a default of `True` though